README.md

```diff
@@ -76,317 +110,607 @@ cartridge --help
-result of running `cartridge pack docker`).
+  ```dockerfile
+  FROM centos:8
+  RUN yum install -y gcc-c++
+  # Note that git, gcc, make, cmake, unzip packages
+  # will be installed anyway
+  ```
 
-Both images are created based on `centos:8`.
-All packages required for the default  `cartridge` application build
-(`git`, `gcc`, `make`, `cmake`, `unzip`) are installed on the build image.
-Opensource Tarantool is installed on both images
-(if Tarantool Enterprise isn't used).
+* `Dockerfile.cartridge`:
 
-If your application requires some other applications for build or runtime, you
-can specify base layers for build and runtime images:
+  ```dockerfile
+  FROM centos:8
+  RUN yum install -y zip
+  ```
 
-* build image: `Dockerfile.build.cartridge` or `--build-from`;
-* runtime image: `Dockerfile.cartridge` or `--from`.
+#### Runtime image tag
 
-The base image dockerfile should be started with the `FROM centos:8` line
-(except comments).
+The runtime image is tagged as follows:
 
-For example, if your application requires `gcc-c++` for build and `zip` for
-runtime:
+* `<name>:<detected_version>`: by default;
+* `<name>:<version>`: if the `--version` parameter is specified;
+* `<tag>`: if the `--tag` parameter is specified.
 
-`Dockerfile.cartridge.build`:
+#### Tarantool Enterprise SDK
 
-```dockerfile
-FROM centos:8
-RUN yum install -y gcc-c++
-# Note, that git, gcc, make, cmake, unzip packages
-# will be installed anyway
-```
+If you use Tarantool Enterprise, you should explicitly specify the Tarantool SDK
+to be delivered on the runtime image.
 
-`Dockerfile.cartridge`:
+If you want to use the SDK from your local machine, just pass the `--sdk-local`
+flag to the `cartridge pack docker` command.
 
-```dockerfile
-FROM centos:8
-RUN yum install -y zip
-```
+Alternatively, you can specify a local path to another SDK using the `--sdk-path`
+option (or the environment variable `TARANTOOL_SDK_PATH`, which has lower priority).
 
-#### Building the app
+#### Customizing the application build in Docker
 
-If you want the `docker build` command to be run with custom arguments, you can
-specify them using the `TARANTOOL_DOCKER_BUILD_ARGS` environment variable.
+You can pass custom arguments for the `docker build` command via the
+`TARANTOOL_DOCKER_BUILD_ARGS` environment variable.
 For example, `TARANTOOL_DOCKER_BUILD_ARGS='--no-cache --quiet'`
 
-#### Using the result image
+#### Using the runtime image
 
-The application code will be placed in the `/usr/share/tarantool/${app_name}`
-directory. An opensource version of Tarantool will be installed to the image.
+The application code is placed in the `/usr/share/tarantool/${app_name}`
+directory. An opensource version of Tarantool is installed to the image.
 
 The run directory is `/var/run/tarantool/${app_name}`,
 the workdir is `/var/lib/tarantool/${app_name}`.
 
+The runtime image also contains the file `/usr/lib/tmpfiles.d/<name>.conf`
+that allows the instance to restart after server restart.
+
```

Comment:
Please, change an example start instance command to

```bash
docker run -d \
                --name instance-1 \
                -e TARANTOOL_INSTANCE_NAME=instance-1 \
                -e TARANTOOL_ADVERTISE_URI=3302 \
                -e TARANTOOL_CLUSTER_COOKIE=secret \
                -e TARANTOOL_HTTP_PORT=8082 \
                -p 127.0.0.1:8082:8082 \
                myapp:1.0.0
```

Just add a `-p` option for ports publishing

Reply:
ok